src/test/java/com/contentful/java/cda/ClientTest.java

```diff
@@ -148,8 +149,10 @@ public void authHeader() throws Exception {
   @Test
   @Enqueue
   public void userAgentHeader() throws Exception {
-    String versionName = Util.getProperty("version.name");
+    String versionName = GeneratedBuildParameters.PROJECT_VERSION;
     assertThat(versionName).isNotEmpty();
+    assertThat(versionName).doesNotContain("${");
+
```

Comment:
could we use a regular expression to test the version number?

Reply:
Yes please!